azure-pipelines/templates/steps/vsts-release.yml

```diff
@@ -0,0 +1,79 @@
+# File: azure-pipelines\templates\steps\nightly-snapshot-releases\vsts-release.yml
+# Description: Template to assemble and publish Snapshot/Latest versions of ${{ parameters.projectName }}
+# Variable: 'ENV_VSTS_MVN_ANDROIDADACCOUNTS_USERNAME' was defined in the Variables tab
+# Variable: 'mvnAccessToken' was defined in the Variables tab
+# Variable: 'folderName' refers to the project folder
+# Variable: 'gradleTasks' list all the gradle tasks to be used.
```

Comment:
Not sure how we pass a list here...and what is the usage?

Reply:
I pass a string(charsequence) in this case gradleTasks can accept: <projectName>: clean projectName:assemble<VARIANT>. The build tasks could be a bit different in the case where java-libraries assembly differ from android assembly as the android plugin has different assemble tasks for each variant.